optout/src/main/java/gov/cms/ab2d/optout/OptOutProcessor.java

```diff
@@ -132,6 +134,33 @@ public String createResponseContent() {
         return responseContent.toString();
     }
 
+    public OptOutResults getOptOutResults() {
+        int totalFromDb = 0;
+        try (var dbConnection = DriverManager.getConnection(parameterStore.getDbHost(), parameterStore.getDbUser(), parameterStore.getDbPassword());
+             var statement = dbConnection.createStatement();
+             ResultSet rs = statement.executeQuery(COUNT_STATEMENT)
+        ) {
+            while (rs.next()) {
+                totalFromDb = rs.getInt("total");
+            }
+
+            int numberOptedIn = 0;
+            int numberOptedOut = 0;
+
+            for (OptOutInformation optOut : optOutInformationList) {
+                if (optOut.getOptOutFlag()) {
+                    numberOptedIn++;
+                } else {
+                    numberOptedOut++;
+                }
```

Comment:
Just double-checking that this isn't the opposite? `getOptOutFlag() == true` means that they're opted-in?

Reply:
getOptOutFlag() == true means "yes, share my data"

Reply:
It is certainly confusing, but I've been assured that:

OptOutFlag 'true' means "I agree to have my data shared".
OptOutFlag 'false' means "I do not want my data shared". 
OptOutFlag 'empty' or 'null' means 'I haven't called in to opt one way or another, so default to sharing my data"